src/popup/router/components/OverflowableText.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <span class="overflowable-text">
+    <span class="text">{{ isName ? isName : text }}</span>
+    <span v-if="isName">{{ `.chain` }}</span>
+  </span>
+</template>
+
+<script>
+export default {
+  props: {
+    text: { type: String, required: true },
+  },
+  computed: {
+    isName() {
+      return this.text.match(/([a-zA-Z0-9]+)\.chain/)?.[1];
```

Comment:
```suggestion
      return this.text.match(/^([a-zA-Z0-9]+)\.chain$/)?.[1];
```
I wouldn't like to specify name structure here, I would check for `.endsWith('.chain')` instead